velox/exec/Task.cpp

```diff
@@ -1625,6 +1636,7 @@ void Task::addOperatorStats(OperatorStats& stats) {
       stats.operatorId <
           taskStats_.pipelineStats[stats.pipelineId].operatorStats.size());
   aggregateOperatorRuntimeStats(stats.runtimeStats);
+  addRunningTimeOperatorMetrics(stats);
```

Comment:
@xiaoxmeng Why do we need this change? addInput/getOutput/finish timing is already reported via OperatorStats.

Reply:
@mbasmanova that's true. It seems that we don't need this @spershin

Reply:
@mbasmanova , @xiaoxmeng 

Where can we see them in query UI and can we see them per thread/driver?

This change was inspired by the lack of visibility on how long a particular operator was in running state. We now have blocked and queued, but to see the whole picture we need 'running' as well.

Let me know if we have access to this info somewhere.